pkg/genericapiserver/options/server_run_options.go

```diff
@@ -71,6 +71,7 @@ type ServerRunOptions struct {
 	EnableSwaggerUI           bool
 	EnableWatchCache          bool
 	EtcdServersOverrides      []string
+	ExpectedClusterSize       int
```

Comment:
This is really really hard to take. apiserver should be *publishing* this value, not accepting it as a command line argument.